src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -672,9 +672,12 @@ void xocl_mm_get_usage_stat(struct xocl_drm *drm_p, u32 ddr,
 		return;
 	}
 
-	mm_stat = xocl_mm->mm_usage_stat[ddr];
-	pstat->memory_usage = mm_stat ? mm_stat->memory_usage : 0;
-	pstat->bo_count = mm_stat ? mm_stat->bo_count : 0;
+	mm_stat = &drm_p->mm_usage_stat[ddr];
+
+	if (mm_stat->is_used) {
```

Comment:
Don't you need to check if mm_stat is NULL and assign 0 to usage and bo_count if it is NULL or not used? The original code does check. And if you check it here, there is no need to check inside this if statement.

Reply:
I missed the line 675, so mm_state will never be NULL. So, please just change the next two lines and check if mm_stat->is_used, instead of checking mm_stat is NULL.

Reply:
Fixed